### ISSUE TITLE
`buffers` only show file currently opened in buffer

### ISSUE THREAD

**Issue by yofriadi** [nvim-telescope/telescope.nvim#850]
it's a question not a bug

### Description
when using `:Telescope buffers` it only show file *currently* opened in buffer, when I already open multiple files before.

**Expected Behavior**
list all opened file and list it in `:Telescope buffers`, not just *currently* opened file in buffer.

**Actual Behavior**
it only list file currently opened in buffer.

for the simplicity reproducible, I only use `:Telescope buffers` and map to my shortcut.
maybe I'm a bit mistaken here on how to do it, but I currently can't find any solution otherwise use other plugin like https://github.com/Yohannfra/Nvim-Switch-Buffer or https://github.com/vim-ctrlspace/vim-ctrlspace (buffer list)

any help is appreciated, thank you
also thank you for the amazing plugin

**Comment by smithbm2316** [comment]
`:Telescope buffers` should show all of the buffers that you have currently opened in Neovim, can you please fill out the template for a bug provided when you created the issue, and run the minimal config to see if you still have the issue?

**Comment by Conni2461** [comment]
There is an option to enable this: `lua require'telescope.builtin'.buffers{ show_all_buffers = true }` or `Telescope buffers show_all_buffers=true`
I know it should be default. Its requested here: https://github.com/nvim-telescope/telescope.nvim/issues/415#issuecomment-764622897

This is also a dup of #368, #338 and #97 and probably more, so i am closing this

**Comment by smithbm2316** [comment]
> There is an option to enable this: `lua require'telescope.builtin'.buffers{ show_all_buffers = true }` or `Telescope buffers show_all_buffers=true`
> I know it should be default. Its requested here: [#415 (comment)](https://github.com/nvim-telescope/telescope.nvim/issues/415#issuecomment-764622897)
> 
> This is also a dup of #368, #338 and #97 and probably more, so i am closing this

@Conni2461 well this is odd. For me, `buffers` acts by default like `show_all_buffers` is set to true (i've searched my config, I've done nothing to add this option myself). However, when using a plugin like `vim-obsession` or a new plugin I've been using called `auto-session`, when my previous session is loaded in Neovim, all of the hidden buffers don't show. But the hidden buffers I have set always have shown without setting that option explicitly, so long as I opened them during _that_ Neovim session. Not sure if that makes sense the distinction, but now I'm curious as to why I've never had any need to set `show_all_buffers` to `true` to see all of my hidden buffers in a normal Neovim session.
